### DISCUSSION TITLE
Ramdisk with 1TB of memory

### DISCUSSION THREAD

**Discussion by koiesmith** [Chia-Network/bladebit#93] [Q&A]
Harold,

I have a few servers with 1TB of memory so I wanted to try to create a ramdisk using some of the extra memory since the process only consumes 419GB.  I created a 300GB ram disk and mounted to /mnt/ram/ and set bladebit to plot to this directory.  I'm getting the following error.

Finished Phase 2 in 23.80 seconds.
Running Phase 3
  Compressing tables 1 and 2...
Fatal Error:
  Failed to write table 1 to disk.

Any ideas?

**Comment by harold-b** [comment]
Well it is evident that it fails once it actually attempts to write the table to disk. I wouldn't know any more without any more context. Not sure why I did not show the error code there, so I will see about being a bit more verbose there.
